index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Cristian-Teddy Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
- It's a good habit to remove comments from your codebase. The author suggests that this comment should be deleted after reading the message. This is because comments make our code look ugly.

Reply:
- [] I think to be more clear, you could ask the  student to remove commented code, because some comments are for descriptive purposes.  It is good be precise about commented code.